boltz_client/boltz.py

```diff
@@ -118,25 +118,41 @@ def check_version(self):
             headers={"Content-Type": "application/json"},
         )
 
+    def add_reverse_swap_fees(self, amount: int) -> int:
+        rev = self.fees["minerFees"]["baseAsset"]["reverse"]
+        fee = rev["claim"] + rev["lockup"]
+        percent = self.fees["percentage"]
+        return ceil((amount + fee) / (1 - (percent / 100)))
+
+    def substract_swap_fees(self, amount: int) -> int:
+        fee = self.fees["minerFees"]["baseAsset"]["normal"]
+        percent = self.fees["percentageSwapIn"]
+        return floor((amount - fee) / (1 + (percent / 100)))
```

Comment:
Things like these should have proper tests to ensure accuracy in edge cases and to avoid regressions

Reply:
https://github.com/BoltzExchange/boltz-client-python/blob/main/tests/test_boltz_lifecycle_reverse_swap.py#L63